### ISSUE TITLE
How can I record phone conversation with the terminal commands or script?

### ISSUE THREAD

**Issue by leftside2** [ubports/ubuntu-touch#1021]
### Description of the feature
Nexus5, Ubuntu Touch 16.04, phone conversation recording problem. https://github.com/ubports/ubuntu-touch/issues/1020
I understood that the RECORD/MIC icon on the phone call screen is not the one I'm expecting but simply a kind of mute-mic feature.
But how can I record the phone conversation via terminal script anyway??

### Illustrations

- Device: Nexus5
- Channel:
- Build: 16.04

### Expected behavior
By simply executing a bash script in the terminal, I might record the conversation between the opposite person who is talking with me on the phone.

**Comment by leftside2** [comment]
@hummlbach , Thanks for ur info, I'll try.

**Comment by leftside2** [comment]
@hummlbach , No. That is just the same app as the default recorder app on the phone when u install ubuntu-touch. It records only my voice not the other one's speech.

**Comment by hummlbach** [comment]
Oh, mhmm then its maybe device spezific, cause this works on turbo. Shouldn't make a difference I guess, but did you call out or did you receive the call? (I have just checked with an outgoing call.)

**Comment by leftside2** [comment]
@hummlbach , only call out. It didn't work like I said on the above.

**Comment by dobey** [comment]
@hummlbach The voice recorder app only can get what it hears on the microphone. The audio stream from the other end of the call is not accessible to it.

**Comment by hummlbach** [comment]
@dobey my turbo does for sure. I have some eye/ear whitnesses ;-) (no loud speaker enabled)

**Comment by tallero** [comment]
I could get decent call recording using the recorder and loud speaker.
